docs/data/data-grid/editing/StartEditButtonGrid.tsx

```diff
@@ -69,40 +111,28 @@ function EditToolbar(props: EditToolbarProps) {
 export default function StartEditButtonGrid() {
   const [selectedCellParams, setSelectedCellParams] =
     React.useState<GridCellParams | null>(null);
+  const [cellModesModel, setCellModesModel] = React.useState<GridCellModesModel>({});
 
   const handleCellClick = React.useCallback((params: GridCellParams) => {
     setSelectedCellParams(params);
   }, []);
 
-  const handleCellEditStart = (
-    params: GridCellParams,
-    event: MuiEvent<React.SyntheticEvent>,
-  ) => {
-    event.defaultMuiPrevented = true;
-  };
-
-  const handleCellEditStop: GridEventListener<GridEvents.cellEditStop> = (
-    params,
-    event,
-  ) => {
-    event.defaultMuiPrevented = true;
-  };
-
   return (
     <div style={{ height: 400, width: '100%' }}>
       <DataGrid
         rows={rows}
         columns={columns}
+        cellModesModel={cellModesModel}
         onCellClick={handleCellClick}
```

Comment:
Not related to this PR, but is there better way to get currently focused cell in `DataGrid`?
`onCellClick` doesn't work with keyboard navigation.

In `DataGridPro` it's possible to use `apiRef.current.subscribeEvent(GridEvents.cellFocusIn)`, but what about `DataGrid`?

Reply:
`componentsProps.cell` can be used to pass `onFocus` to the cell. This demo never worked with keyboard. I updated it to use `onFocus` instead.